r/tools/nixlibs.R

```diff
@@ -47,7 +47,10 @@ quietly <- !env_is("ARROW_R_DEV", "true")
 
 # Default is build from source, not download a binary
 build_ok <- !env_is("LIBARROW_BUILD", "false")
-binary_ok <- env_is("LIBARROW_BINARY", "true") || env_is("NOT_CRAN", "true")
+# For LIBARROW_BINARY we support "true" or the name of the OS to use to
+# locate the appropriate binary (e.g., ubuntu-18.04). When NOT_CRAN=true, the
+# configure script sets LIBARROW_BINARY=true.
```

Comment:
```suggestion
# locate the appropriate binary (e.g., ubuntu-18.04). When NOT_CRAN=true, and
# LIBARROW_BINARY is unset, the configure script sets LIBARROW_BINARY=true.
```

A very minor nit but I did just confirm that `NOT_CRAN=TRUE LIBARROW_BINARY="ubuntu-18.04"` should work and not change `LIBARROW_BINARY` to true